draft-ietf-quic-transport.md

```diff
@@ -5113,6 +5113,8 @@ discussed in {{?QUIC-MANAGEABILITY=I-D.ietf-quic-manageability}}.
 
 The spin bit is an OPTIONAL feature of this version of QUIC. A QUIC stack that
 chooses to support the spin bit MUST implement it as specified in this section.
+A QUIC stack that chooses not to support the spin bit MUST behave as if it were
```

Comment:
Do we need to add this sentence?

Reply:
I have a simpler change:

> The spin bit is an OPTIONAL feature of this version of QUIC.   An endpoint that does not support this feature MUST disable it, as defined below.

Reply:
Use of "QUIC stack" is also odd. I like @martinthomson 's simpler suggestion.